app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    rating_sum = 0
+    for player in players:
+        rating_sum += player.get_rating()
+    return rating_sum
+
+
+def elves_concert(elfs: list[Elf]) -> None:
+    for elf in elfs:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list[Dwarf]) -> None:
```

Comment:
The type annotation for the `dwarves` parameter should be `list[Dwarf]` instead of `list[[Dwarf]]`. This ensures proper type hinting.